### PR TITLE
feat: Ask for npm OTP token in the terminal at the point of publishing the package

### LINKED ISSUE #191: Ask for npm OTP token in the terminal at the point of publishing the package
### Prerequisites

- [X] I have written a descriptive issue title
- [X] I have searched existing issues to ensure the feature has not already been requested

### 🚀 Feature Proposal

ask a user to enter the 6 digit TOTP code at runtime of the CLI

### Motivation

at the moment you have to input yout TOTP token upfront as CLI parameter `--npm-otp 123456`. This is not a good solution because this code changes every 30 seconds. 
If operations of releasify take more than that user cannot use releasify, because it would always fail with 
```
[] ERROR: npm publish returned code 1 and signal null 
    Error: npm publish returned code 1 and signal null 
        at ChildProcess.<anonymous> (/home/capaj/.nvm/versions/node/v16.7.0/lib/node_modules/releasify/lib/npm.js:22:14)
        at ChildProcess.emit (node:events:394:28)
        at maybeClose (node:internal/child_process:1064:16)
        at Socket.<anonymous> (node:internal/child_process:450:11)
        at Socket.emit (node:events:394:28)
        at Pipe.<anonymous> (node:net:672:12)

```
This is not a typical usecase, but it could occur on slower connections

### Example

_No response_

### PR DESCRIPTION
**Pull request by capaj**
fixes #191

#### Checklist

- [ ] run `npm run test` and `npm run benchmark`
- [ ] tests and/or benchmarks are included
- [ ] documentation is changed or added
- [ ] commit message and code follows the [Developer's Certification of Origin](https://github.com/fastify/.github/blob/master/CONTRIBUTING.md#developers-certificate-of-origin-11)
      and the [Code of conduct](https://github.com/fastify/.github/blob/master/CODE_OF_CONDUCT.md)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -72,7 +73,6 @@ releasify publish [--path|-p <path>]             ➡ The path to the project to
                   [--gh-group-by-label|-l <label>] ➡ Group the commits in the changelog message by PR's labels
                   [--npm-access|-a <string>]       ➡ It will set the --access flag of `npm publish` command. Useful for scoped modules. The value must be [public, restricted]
                   [--npm-dist-tag <string>]        ➡ It will add a npm tag to the module, like `beta` or `next`
-                  [--npm-otp <code>]               ➡ It will provide the otp code to the npm publish
```
Comment (mcollina) [inline README.md:75]: please keep this property

Comment (capaj) [reply README.md:75]: oh yeah I wanted to comment here just before the standup started that we are probably using this on some CIs, will retain this

Comment (Eomm) [inline README.md:75]: Could you leave this argument?

If wee keep it, we could still think to use releasify automation within optic https://github.com/nearform/optic#usage

Comment (capaj) [reply README.md:75]: will do

**Pull request closed without merging**